src/utils/webrtc/simplewebrtc/simplewebrtc.js

```diff
@@ -123,6 +123,10 @@ function SimpleWebRTC(opts) {
 			// "nickChanged" can be received from a participant without a Peer
 			// object if that participant is not sending audio nor video.
 			self.emit('nick', { id: message.from, name: message.payload.name })
+		} else if (message.type === 'raiseHand') {
+			// "raisedHand" can be received from a participant without a Peer
+			// object if that participant is not sending audio nor video.
+			self.emit('raisedHand', { id: message.from, raised: message.payload })
```

Comment:
ah no, you moved it here. still, I now find it strange that we need to modify the simplewebrtc lib for any new messages/signals